platform/vs/syncd-vs.mk

```diff
@@ -3,13 +3,6 @@ $(LIBSAIREDIS)_DEB_BUILD_PROFILES += syncd vs
 SYNCD_VS = syncd-vs_1.0.0_$(CONFIGURED_ARCH).deb
 $(SYNCD_VS)_RDEPENDS += $(LIBSAIREDIS) $(LIBSAIMETADATA) $(LIBSAIVS)
 
-ifeq ($(BLDENV),bookworm)
```

Comment:
Why remove dependency on DASH SAI?

Reply:
Good catch! This was a change I had made locally to speed up my local build and didn't mean to commit it. Removed.